addon/internal-session.js

```diff
@@ -6,7 +6,6 @@ const { RSVP, on } = Ember;
 export default Ember.ObjectProxy.extend(Ember.Evented, {
   authenticator:       null,
   store:               null,
-  container:           null,
   isAuthenticated:     false,
   attemptedTransition: null,
   content:             { authenticated: {} },
```

Comment:
The problem is actually this. This `content` object will be shared across all instances of the `InternalSession` class as it is defined on the prototype…

This would have to be replaced with 

```js
_init() {
    this._super(...arguments);
    this.set('content', { authenticated: {} });
  },
```

which might cause other problems with other tests but this is actually a sever bug - not sure what I was thinking when adding this…